gcovr/tests/filter-test2/Makefile

```diff
@@ -2,8 +2,7 @@ CFLAGS= -fprofile-arcs -ftest-coverage -fPIC
 
 BASE_OS:=$(shell uname | cut -d'-' -f1)
 ifeq ($(BASE_OS),MSYS_NT)
-  #GCOVR_TEST_OPTIONS := -f $(shell cygpath -w "$$PWD")\main.cpp
-  GCOVR_TEST_OPTIONS := -f main.cpp
+  GCOVR_TEST_OPTIONS := -f $(shell cygpath -w "$$PWD")\main.cpp
```

Comment:
Absolute filters are known to not work on Windows right now. I would rather have this test run with a relative filter on Windows than skipping it entirely.

Can you explain how this change is related to collecting coverage or to extracting `__main__`? If there is a need for this commit, OK. I'd otherwise prefer to not merge it.

Reply:
the rebase was a bit to extensive... I had no preference between running a relative filter or skipping the test. I will re-enable the test as proposed.

Reply:
PS: the test was not passing the correct options anyway before those commits. It worked with python 2.7 but failed with python 3.6 now that tests are run with the correct interpreter on windows.